src/Workspaces/Core/Portable/Rename/ConflictEngine/ConflictResolver.Session.cs

```diff
@@ -30,14 +30,14 @@ internal static partial class ConflictResolver
         private class Session
         {
             // Set of All Locations that will be renamed (does not include non-reference locations that need to be checked for conflicts)
-            private readonly HeavyweightRenameLocations _renameLocationSet;
+            private readonly SymbolicRenameLocations _renameLocationSet;
```

Comment:
HeavyweightRenameLocations  renamed to  SymbolicRenameLocations

Reply:
Love this renamed type. Did you end up changing the lightweight version as well?

Reply:
nope.  i don't have a great name for it.  any suggestions?